tensorflow/contrib/tensorrt/log/trt_logger.cc

```diff
@@ -12,8 +12,6 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#if GOOGLE_CUDA
```

Comment:
@samikama why removing this? I think it is required as if trt is not enabled we should not include the logger code.

Reply:
The code is not compiled with necessary macros. Moreover this rule should not trigger if trt is not enabled at all. Am I missing something?

Reply:
Just for my understanding, I though trt_logger is not compiled if trt is not enabled due to bazel rules. Is this not the case?

Reply:
Ok. Put it back. It works now since you changed the library to be a cuda library.

Reply:
You're right. The logger is not compiled when trt is not enabled, but we still want it to be compilable (i.e. if I manually compile it it should not produce error) in that case. Thanks for adding back. :)